### PR TITLE
add keras2 LocallyConnected1D&Cropping1D&BatchNormalization layers

### PR DESCRIPTION
**Pull request by yjjiang11**

### CODE REVIEW

**Comment by yjjiang11** [comment]
Jenkins passed:http://172.168.2.101:8080/job/ZOO-PR-Validation-Other/786/

**Comment by yjjiang11** [comment]
Jenkins passed:http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/784/ @hkvision

File: pyzoo/zoo/pipeline/api/keras2/layers/local.py [hunk 1]
```diff
@@ -14,7 +14,64 @@
...
     long = int
     unicode = str
+
+
+class LocallyConnected1D(ZooKeras2Layer):
+    """
+    Locally-connected layer for 1D inputs which works similarly to the TemporalConvolution
+    layer, except that weights are unshared, that is, a different set of filters is applied
+    at each different patch of the input..
+    Border mode currently supported for this layer is 'valid'.
```
Comment (yjjiang11) [inline pyzoo/zoo/pipeline/api/keras2/layers/local.py]: padding
